app/javascript/mastodon/features/status/index.jsx

```diff
@@ -70,6 +70,9 @@ const messages = defineMessages({
   redraftMessage: { id: 'confirmations.redraft.message', defaultMessage: 'Are you sure you want to delete this status and re-draft it? Favourites and boosts will be lost, and replies to the original post will be orphaned.' },
   revealAll: { id: 'status.show_more_all', defaultMessage: 'Show more for all' },
   hideAll: { id: 'status.show_less_all', defaultMessage: 'Show less for all' },
+  statusTitleWithText: { id: 'status.title.with_text', defaultMessage: '"{truncatedStatusText}" by {user}' },
+  statusTitleWithoutText: { id: 'status.title.without_text', defaultMessage: 'Post with {attachmentCount, plural, one {one attachment} other {{attachmentCount} attachments}} by {user}' },
+  ellipsis: { id: 'punctuation.ellipsis', defaultMessage: '…' },
```

Comment:
I'm not convinced by the new format, I think “user: truncated post text - site name” was fine…

For consistency, the format for text-less posts could be “user posted N media attachments - site name” for consistency.

Reply:
@ClearlyClaire, I implemented the title like this, so it's more human readable and adaptable to the server's or user's locale. To see what I mean, look at the tab title for this PR and notice how it has a similar structure to the status title I implemented. Also, I decided to implement the ellipsis as a separate translation because it can be written differently in different languages.